vortex-array/src/array/varbin/compute/compare.rs

```diff
@@ -20,6 +24,40 @@ impl CompareFn<VarBinArray> for VarBinEncoding {
             let nullable = lhs.dtype().is_nullable() || rhs_const.dtype().is_nullable();
             let len = lhs.len();
 
+            if matches!(operator, Operator::Eq | Operator::NotEq) {
```

Comment:
Can you do all operators.

a < "" iff false
a < "" iff a == ""
a > "" if a != ""
a >= "" true